src/main/java/ca/wolfram/beta/symath/operations/DivideNode.java

```diff
@@ -0,0 +1,42 @@
+package ca.wolfram.beta.symath.operations;
+
+import ca.wolfram.beta.symath.MathNode;
+import ca.wolfram.beta.symath.MathUtils;
+import ca.wolfram.beta.symath.NodeType;
+import ca.wolfram.beta.symath.base.BaseNode;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Allan Wang on 2017-05-11.
+ */
+public class DivideNode {
```

Comment:
what i meant by this comment is to not have NodeType.DIVIDE. otherwise say in the future, i do a switch case on all the operation. and i code in some logic for `case DIVIDE` that logic will always be unused